### ISSUE TITLE
Dark mode gallery issue

### ISSUE THREAD

**Issue by pipedreambomb** [expo/google-fonts#37]
![Screenshot 2021-07-03 19 55 30](https://user-images.githubusercontent.com/677979/124364343-d123dd80-dc38-11eb-9dc7-66b0ebbdcb29.png)

I think it's really cool that you've integrated a gallery into the GitHub repo. The only unfortunate thing is, when you're browsing GitHub in dark mode, it's impossible to make out the fonts as they're all in black!

**Comment by ccheever** [comment]
Yes! Not sure what the best way to fix this is. Possibly using white as the background for those images. That messes up the tabe's natural alternating color formatting, which is maybe OK.
